### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by albertoT75**
please review

### CODE REVIEW

**Comment by albertoT75** [comment]
Please review

**Comment by albertoT75** [comment]
removed new and create method for kindergarden

File: app/controllers/kindergardens_controller.rb [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
   end
 
+
+
+
+
+  private
+
+  def kindergarden_params
+    params.require(:kindergarden).permit(:name, :address, :language, :capacity)
```
Comment (0math) [inline app/controllers/kindergardens_controller.rb]: need also add :photo, :email, :phone

Comment (albertoT75) [reply app/controllers/kindergardens_controller.rb]: added params: photo, email and phone for Kindergarden model

**Pull request merged**